core/store/src/main/java/mil/nga/giat/geowave/core/store/operations/config/AddStoreCommand.java

```diff
@@ -15,14 +17,17 @@
 
 import mil.nga.giat.geowave.core.cli.annotations.GeowaveOperation;
 import mil.nga.giat.geowave.core.cli.api.Command;
+import mil.nga.giat.geowave.core.cli.api.DefaultOperation;
 import mil.nga.giat.geowave.core.cli.api.OperationParams;
 import mil.nga.giat.geowave.core.cli.operations.config.ConfigSection;
 import mil.nga.giat.geowave.core.cli.operations.config.options.ConfigOptions;
+import mil.nga.giat.geowave.core.cli.parser.ManualOperationParams;
 import mil.nga.giat.geowave.core.store.operations.remote.options.DataStorePluginOptions;
 
 @GeowaveOperation(name = "addstore", parentOperation = ConfigSection.class)
 @Parameters(commandDescription = "Create a store within Geowave")
-public class AddStoreCommand implements
+public class AddStoreCommand extends
+            DefaultOperation implements
```

Comment:
Instead of extending DefaultOperation, I think would be fine just extending ServerResource.  It seems DefaultOperation really only does 3 things - it implements Operation, provides an empty implementation of Operation's prepare function, and extends ServerResource.  The third one is needed, but the first two are unnecessary since AddStoreCommand originally only implements Command, which does not require the prepare function.